<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrustedClientsAddressesDialog</class>
 <widget class="QDialog" name="TrustedClientsAddressesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trusted clients addresses</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="2" column="1">
    <widget class="QToolButton" name="removeButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../resources/vsedit-job-server-watcher.qrc">
       <normaloff>:/delete.png</normaloff>:/delete.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="addButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../resources/vsedit-job-server-watcher.qrc">
       <normaloff>:/add.png</normaloff>:/add.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="warningLabel">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 0, 0);
font-size: 24px;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>WARNING! Adding any address to this list opens your PC for a full control and makes it vulnerable for potential hacker attack!</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="addressEdit"/>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>308</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" rowspan="2">
    <widget class="QListWidget" name="addressesList"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/vsedit-job-server-watcher.qrc"/>
 </resources>
 <connections/>
</ui>
